tools/auto-fuzz/manager.py

```diff
@@ -19,35 +19,46 @@
 import shutil
 import subprocess
 import constants
+import requests
 import shlex
+import tarfile
 import threading
 try:
     import tqdm
 except ImportError:
     print("No tqdm module, skipping progress bar")
 
+# temporary fix: adding fuzz-introspector to the system path
+sys.path.insert(0, '../../src/')
 from fuzz_introspector import commands
 
 # Auto-fuzz modules
 import base_files
 import oss_fuzz_manager
 import fuzz_driver_generation_python
+import fuzz_driver_generation_jvm
 
 from typing import List, Any
 from multiprocessing.dummy import Pool as ThreadPool
 
 # Set default directories and error if they do not exist
+tqdm_tracker = None
 basedir = os.path.dirname(os.path.realpath(__file__))
-FUZZ_INTRO_BASE = basedir + "/../../"
+FUZZ_INTRO_BASE = basedir + "/../../../fuzz-introspector"
```

Comment:
I don't think we should change this? considering the current location of `auto-fuzz` in `tools/auto-fuzz`

Reply:
Changed.